### DISCUSSION TITLE
Double Knock mode

### DISCUSSION THREAD

**Discussion by richardbrn** [MiguelRipoll23/homebridge-securitysystem#246] [Q&A] [answered]
Hello - just got into Homebridge and setting up the security system.  Very impressed with it!
Could someone explain what the double knock mode is for?
Thanks

**Comment by MiguelRipoll23** [comment, marked as answer]
Hello,
When the double knock option is enabled, the sensor must trigger the security system twice within a short time window in order to the siren to go off. Used to avoid accidental triggers.

**Comment by richardbrn** [comment]
Thank you


Sent from my iPhone...

On 11 Jul 2022, at 18:08, Miguel Ripoll ***@***.***> wrote:

﻿

This message originated from outside of NHSmail. Please do not click links or open attachments unless you recognise the sender and know the content is safe.

Hello,
When the double knock option is enabled, the sensor must trigger the security system twice within a short time window in order to the siren to go off. Used to avoid accidental triggers.

—
Reply to this email directly, view it on GitHub<https://github.com/MiguelRipoll23/homebridge-securitysystem/discussions/246#discussioncomment-3123186>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AR433MY5LWQ7HBKWDCCZYJLVTRIGHANCNFSM53EYKAPA>.
You are receiving this because you authored the thread.Message ID: ***@***.***>


************************************************************************************** ******************************

This message may contain confidential information. If you are not the intended recipient please:
i) inform the sender that you have received the message in error before deleting it; and
ii) do not disclose, copy or distribute information in this e-mail or take any action in relation to its content (to do so is strictly prohibited and may be unlawful).
Thank you for your co-operation.

NHSmail is the secure email, collaboration and directory service available for all NHS staff in England. NHSmail is approved for exchanging patient data and other sensitive information with NHSmail and other accredited email services.

For more information and to find out how you can switch visit Joining NHSmail – NHSmail Support<https://support.nhs.net/article-categories/joining-nhsmail/>
